.github/workflows/import-test.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     # Check https://hub.docker.com/r/pytorch/pytorch/tags for latest tags
     container:
-      image: pytorch/pytorch:1.11.0-cuda11.3-cudnn8-runtime
+      image: pytorch/pytorch:1.12.1-cuda11.3-cudnn8-runtime
```

Comment:
revert.

Reply:
The previous torch version was breaking import checks and Somshubra advised to change version, which fixed the problem. Do u think it is better to revert?

Reply:
I would suggest keeping as-is. If `1.11.0` really hurts, then it'd better to file a separate PR to upgrade version, just like what this https://github.com/NVIDIA/NeMo/pull/4273/files did.